test/services/user/services/AdminUsers.test.js

```diff
@@ -668,7 +668,7 @@ describe('AdminUsersService', () => {
 		const studentData = {
 			firstName: 'testDeleteStudent',
 			lastName: 'lastTestDeleteStudent',
-			email: 'testDeleteStudent@de.de',
+			email: `testDeleteStudent${Date.now()}@de.de`,
```

Comment:
Maybe not in scope, but we can reduce the passed data, a generic email name and so on are set by testObjects, student is set as default, also we only must pass the schoolId if it is not the default schoolId. Data for test are saved on const student = ... and the test should not look to firstName lastName.

But i am happy about the change that you make. :)